Makefile

```
@@ -156,6 +156,12 @@ toc-update: mdtoc
 toc-verify: mdtoc
 	./hack/verify-toc.sh
 
+.PHONY: helm-verify
+helm-verify: helm ## run helm template and detect any rendering failures
+# test default values
+	$(HELM) template charts/kueue
```

To silence output from helm, maybe like this?

```suggestion
	$(HELM) template charts/kueue  > /dev/null
```

---

Error logs should be still here.

---

Sounds good. Added.